packages/calcite-components/src/components/card/card.e2e.ts

```diff
@@ -116,4 +116,117 @@ describe("calcite-card", () => {
+      themed(
+        html`<calcite-card>
+          <span slot="heading">Heading</span>
+          <span slot="description">Description</span>
+        </calcite-card>`,
+        {
+          "--calcite-card-background-color": {
+            shadowSelector: `.${CSS.contentWrapper}`,
+            targetProp: "backgroundColor",
+          },
+          "--calcite-card-corner-radius": {
+            shadowSelector: `.${CSS.contentWrapper}`,
+            targetProp: "borderRadius",
+          },
+          "--calcite-card-border-color": {
+            shadowSelector: `.${CSS.contentWrapper}`,
+            targetProp: "borderColor",
+          },
+          "--calcite-card-shadow": {
+            shadowSelector: `.${CSS.contentWrapper}`,
+            targetProp: "boxShadow",
+          },
+        },
+      );
+    });
+
+    describe("selectable", () => {
+      themed(
+        html`<calcite-card selection-mode="single">
+          <span slot="heading">Heading</span>
+          <span slot="description">Description</span>
+        </calcite-card>`,
+        {
+          "--calcite-card-selection-icon-color": {
+            shadowSelector: `calcite-icon`,
+            targetProp: "--calcite-ui-icon-color",
+          },
+          "--calcite-card-selection-background-color": {
+            shadowSelector: `.${CSS.checkboxWrapper}`,
+            targetProp: "backgroundColor",
+          },
+          "--calcite-card-selection-background-color-active": {
+            shadowSelector: `.${CSS.checkboxWrapper}`,
+            targetProp: "backgroundColor",
+            state: { press: { attribute: "class", value: CSS.checkboxWrapper } },
+          },
+          "--calcite-card-selection-background-color-hover": {
+            shadowSelector: `.${CSS.checkboxWrapper}`,
+            targetProp: "backgroundColor",
+            state: "hover",
+          },
+        },
+      );
+    });
+
+    describe("selected", () => {
+      themed(
+        html`<calcite-card selected selection-mode="single">
+          <span slot="heading">Heading</span>
+          <span slot="description">Description</span>
+        </calcite-card>`,
+        {
+          "--calcite-card-accent-color-selected": {
+            shadowSelector: `.${CSS.contentWrapper}`,
+            targetProp: "boxShadow",
+          },
+          "--calcite-card-border-color": {
+            shadowSelector: `.${CSS.contentWrapper}`,
+            targetProp: "borderColor",
+          },
+          "--calcite-card-selection-background-color-selected": {
+            shadowSelector: `.${CSS.checkboxWrapper}`,
+            targetProp: "backgroundColor",
+          },
+          "--calcite-card-selection-icon-color-selected": {
+            shadowSelector: `calcite-icon`,
+            targetProp: "--calcite-ui-icon-color",
+          },
+          "--calcite-card-selection-icon-color-hover": {
+            shadowSelector: `calcite-icon`,
+            targetProp: "--calcite-ui-icon-color",
```

Comment:
this should be `--calcite-icon-color` as well as on line :198

Reply:
Thanks for the catch! 🎣 Will fix!